### ISSUE TITLE
Training speed is too slow

### ISSUE THREAD

**Issue by PangzeCheung** [ZhaoJ9014/face.evoLVe.PyTorch#111]
I got 1.3s/it+ when training. My bacth size is 400 and use 4 GPU and disabled "dispaly training loss & acc every DISP_FREQ".
I think 1.3s/it is too slow. Is there a solution？Thank you very much！
我的训练速度只有1.3s/it （tqbm显示）使用了4块1080Ti GPU，而且已经关闭了"dispaly training loss & acc every DISP_FREQ"（即每个iter更新准确率的功能）。但是仍然速度很慢只有1.3s/it。想问一下是我的操作有问题吗？有什么解决方法吗？谢谢您～

**Comment by olivia1823** [comment]
> I got 1.3s/it+ when training. My bacth size is 400 and use 4 GPU and disabled "dispaly training loss & acc every DISP_FREQ".
> I think 1.3s/it is too slow. Is there a solution？Thank you very much！
> 我的训练速度只有1.3s/it （tqbm显示）使用了4块1080Ti GPU，而且已经关闭了"dispaly training loss & acc every DISP_FREQ"（即每个iter更新准确率的功能）。但是仍然速度很慢只有1.3s/it。想问一下是我的操作有问题吗？有什么解决方法吗？谢谢您～

你好，请问你使用face.evolve的代码在webface上测试时达到预期效果了么，我按照论文的参数设置，loss一直没有降下来

**Comment by konioyxgq** [comment]
Hello, have you solved the problem of slow training?

**Comment by sstzal** [comment]
> I got 1.3s/it+ when training. My bacth size is 400 and use 4 GPU and disabled "dispaly training loss & acc every DISP_FREQ".
> I think 1.3s/it is too slow. Is there a solution？Thank you very much！
> 我的训练速度只有1.3s/it （tqbm显示）使用了4块1080Ti GPU，而且已经关闭了"dispaly training loss & acc every DISP_FREQ"（即每个iter更新准确率的功能）。但是仍然速度很慢只有1.3s/it。想问一下是我的操作有问题吗？有什么解决方法吗？谢谢您～

yes, training slow.
I have the same problem with you , and i haven't solve this.  Is this perhaps the case? or have you solved this proble?
